transcom/mymove
# MAIN B-22122

[INT PR #1](https://github.com/transcom/mymove/pull/14417)
[INT PR #2](https://github.com/transcom/mymove/pull/14453)
[INT PR #3](https://github.com/transcom/mymove/pull/14490)

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-22122)

## Summary

Adding AK zips to `zip3_distances` so we can use them to get... distances?

### How to test

1. Fire up your server
2. Should see the migration successfully run
3. In DBeaver query the `zip3_distances` table `WHERE to_zip3 = 999` 
4. Also check ZIPs `998`, `997`, `996`, `995` and any other AK ZIPs you find
